TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.cpp

```diff
@@ -39,6 +39,22 @@ ObjectsHandler::ObjectsHandler(sol::state* lua, sol::table & parent) :
 	*/
 	m_table_objects.set_function(ScriptReserved_GetStaticByName, &ObjectsHandler::GetByName<Static, ScriptReserved_Static>, this);
 
+	/***
+	Get moveables by its slot.
+	@function GetMoveablesBySlot
+	@tparam string slot the unique slot of the Moveable like ANIMATING1
+	@treturn table of Moveables referencing the given slot.
+	*/
+	m_table_objects.set_function(ScriptReserved_GetMoveablesBySlot, &ObjectsHandler::GetMoveablesBySlot<Moveable, ScriptReserved_Moveable>, this);
```

Comment:
You can omit the `ScriptReserved_Moveable` template argument as it doesn't get used

Reply:
done